### DISCUSSION TITLE
How to add directories to be processed?

### DISCUSSION THREAD

**Discussion by q962** [AppImage/AppImageKit#1243] [Q&A] [answered]
I observed that linuxdeploy only processed `/AppDir/usr/bin `, `/AppDir/usr/lib `, `/AppDir/usr/lib/x86_64-linux-gnu/` directory.

How do I add other directories?

**Comment by q962** [comment]
Sorry, I replied in the wrong place.
I should ask questions in linuxdeploy discussions.

**Comment by TheAssassin** [comment, marked as answer]
Depends what "processed" means for you. There is `--deploy-deps-only <your directory>`.

**Reply by q962** [reply]
Handle dependent copies of elf files in the specified directory, set rpath

For example `/AppDir/usr/libexec/`

**Reply by TheAssassin** [reply]
That's exactly what this flag does. If needed, you can `export LD_LIBRARY_PATH=...` to directories inside and outside your AppDir to make sure linuxdeploy finds missing dependencies.
